src/Apps/Artist/Routes/Consign/Components/__tests__/ArtistConsignMeta.test.tsx

```diff
@@ -0,0 +1,99 @@
+import { MockBoot } from "DevTools"
+import { mount } from "enzyme"
+import React from "react"
+import { ArtistConsignMeta } from "../ArtistConsignMeta"
+
+jest.mock("Utils/getENV", () => ({
+  getENV: () => "https://artsy.net",
+}))
+
+describe("ArtistConsignMeta", () => {
+  const props = {
+    artistName: "Alex Katz",
+    artistHref: "/artist/alex-katz",
+    imageURL: "path/to/image.jpg",
+  }
+  const getWrapper = (passedProps = {}) => {
+    return mount(
+      <MockBoot>
```

Comment:
Useful for testing stuff that normally boots inside of apps (all React context-related stuff) -- also needed to test responsive layouts around the `<Media>` component.